ScoutSuite/providers/aws/metadata.json

```diff
@@ -278,18 +278,12 @@
         "elb": {
             "resources": {
                 "elbs": {
-                    "api_call": "describe_load_balancers",
-                    "response": "LoadBalancerDescriptions",
                     "cols": 2,
                     "path": "services.elb.regions.id.vpcs.id.elbs",
                     "callbacks": [
                         [ "match_security_groups_and_resources_callback", {"status_path": ["Scheme"], "sg_list_attribute_name": ["security_groups"], "sg_id_attribute_name": "GroupId"} ],
                         [ "get_lb_attack_surface", {} ]
                     ]
-                },
-                "elb_policies": {
-                    "cols": 2,
-                    "path": "services.elb.regions.id.elb_policies"
```

Comment:
Doesn't this remove the elb policies page from the report?

Reply:
Yes. I grepped `elb_policies` in ScoutSuite and even though they were fetched and parsed (fetched in the old `aws/services/elb.py` file and parsed in the `parse_elb_policies_callback` method defined in `aws/provider.py`), they were not used anywhere, neither for findings nor for an attack_surface analysis. So It looked like dead code to me and  I took the decision to get rid of them. What do you think about that @j4v ?

Reply:
That's fair!